app/src/main/java/org/mozilla/fenix/settings/CustomizationFragment.kt

```diff
@@ -85,9 +84,9 @@ class CustomizationFragment : PreferenceFragmentCompat() {
     private fun bindDarkTheme() {
         radioDarkTheme = requirePreference(R.string.pref_key_dark_theme)
         radioDarkTheme.onClickListener {
-            requireContext().components.analytics.metrics.track(
-                Event.DarkThemeSelected(
-                    Event.DarkThemeSelected.Source.SETTINGS
+            AppTheme.darkThemeSelected.record(
+                AppTheme.DarkThemeSelectedExtra(
+                    "settings"
```

Comment:
We should try to preserve all the values previously set for extra keys, including the letter case. This would mean here we should register "SETTINGS". This is also the value mentioned in the extra key description in `metrics.yaml` file.